src/Message/ImportCommandCreator.php

```diff
@@ -0,0 +1,18 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Message;
+
+use App\MessageHandler\BitbucketImportCommandHandler;
+use App\MessageHandler\GithubImportCommandHandler;
+
+class ImportCommandCreator {
+    public function create($username, $provider) {
+        if ($provider == 'github') {

```

Comment:
better to compare strictly by '==='    thanks to this not only value is compared but also type

Reply:
Fixed